### DISCUSSION TITLE
Genshin Impact whitelisting

### DISCUSSION THREAD

**Discussion by Lombaxtard** [nefarius/HidHide#132] [Q&A]
Genshin Impact (modified Unity engine) has native DirectInput-based support for DualShock 4 (the one I use currently). When trying to whitelist the game's executable DS4 controller is not seen by the game. Turning off HIDHide and restarting the game makes the DS4 appear in the game menu again. It appears that this game somehow uses some other executable to pull HID info from, one I'm not aware of (I tried whitelisting every process that is in the same tree as "GenshinImpact.exe" to no avail). If anyone ever experienced this issue with this game or had a similar case with something else and have found a solution I'd really like to hear it. It's the only game I've had issues with using HIDHide

**Comment by Bakaba** [comment]
I have the same issue. I tried every exe and dll present in the game directory but nothing seems to work; Controller isn't detected by Genshin.

It's certainly Genshin's fault, but it would be nice to have a workaround, I hate to have to open hidhide to deactivate it every time I want to play Genshin
